modules/api/src/test/java/io/fluo/api/config/FluoConfigurationTest.java

```diff
@@ -235,11 +246,17 @@ public void testObserverConfig() {
     Assert.assertEquals(1, ocList.size());
     Assert.assertEquals("Class", ocList.get(0).getClassName());
     Assert.assertEquals(0, ocList.get(0).getParameters().size());
-    
-    config = new FluoConfiguration();
+  }
+  
```

Comment:
Trying to think how we could test each set of props that have the same type (like positive int props, non empty string props).  The best I could think of is using reflection.   

```java
String[] positiveIntMethods = ...;
for(String piProp : positiveIntMethods){
    //run positive int test using refelection
}

String[] nonEmptyMethods = ...;
for(String neProp : nonEmptyMethods){
    //run non empty test using refelection
}

```